packages/shared/components/Profile.svelte

```diff
@@ -28,20 +30,38 @@
             return letters[0] + letters[letters.length - 1]
         }
     }
+
+    let strongholdTooltipAnchor = undefined
+    let showStrongholdTooltip = false
+    function toggleTooltip(): void {
+        showStrongholdTooltip = !showStrongholdTooltip
+    }
 </script>
 
 <div class="flex items-center justify-center w-24">
     <div class="flex flex-col justify-between items-center space-y-3">
-        <div
-            on:click={() => handleOnClick()}
-            class="h-20 w-20 {bgColor
-                ? `bg-${bgColor}-500`
-                : ''} rounded-full font-bold text-center flex items-center justify-center {classes}"
-        >
-            {#if slots}
-                <slot />
-            {:else}
-                <Text type="h3" classes="text-white">{getInitials()}</Text>
+        <div class="relative flex flex-col">
+            <div
+                on:click={() => handleOnClick()}
+                class="h-20 w-20 {bgColor
+                    ? `bg-${bgColor}-500`
+                    : ''} rounded-full font-bold text-center flex items-center justify-center {classes}"
+            >
+                {#if slots}
+                    <slot />
+                {:else}
+                    <Text type="h3" classes="text-white">{getInitials()}</Text>
+                {/if}
+            </div>
+            {#if strongholdVersion && isStrongholdOutdated(strongholdVersion)}
```

Comment:
There are also ledger profiles that will display the error because they will not have a a, you should ideally check that this profile is a software profile